app/components/support_interface/application_choice_component.rb

```diff
@@ -42,7 +42,11 @@ def rows
       rows << { key: 'Reject by default at', value: application_choice.reject_by_default_at.to_s(:govuk_date_and_time) } if application_choice.reject_by_default_at && application_choice.awaiting_provider_decision?
       rows << { key: 'Decline by default at', value: application_choice.decline_by_default_at.to_s(:govuk_date_and_time) } if application_choice.decline_by_default_at && application_choice.offer?
 
-      rows.flatten
+      if ApplicationStateChange::STATES_VISIBLE_TO_PROVIDER.include?(application_choice.status.to_sym)
+        rows << { key: 'API', value: render(SupportInterface::ApplicationAPIRepresentationComponent.new(application_choice: application_choice)) }
+      end
```

Comment:
For extra unmysteriousing, we could add a row here to explain that this isn't visible to the API because it's not in a visible state.

Reply:
Better yet!